src/vuestic-theme/vuestic-components/vuestic-file-upload/VuesticFileUploadGalleryItem.vue

```diff
@@ -67,15 +68,30 @@
       },
       convertToImg () {
         const reader = new FileReader()
+        const imageFileTypes = ['/png', '/jpg', '/jpeg', '/gif']
         reader.readAsDataURL(this.file.image)
         reader.onload = (e) => {
-          this.previewImage = e.target.result
+          for (let i = 0; i < imageFileTypes.length; i++) {
+            if (e.target.result.indexOf(imageFileTypes[i]) >= 0) {
+              this.previewImage = e.target.result
+            }
+          }
+          if (!this.previewImage) {
+            this.isNotImage = true
```

Comment:
Could be a computed prop:
```
computed: {
  isImage () {
    return !!this.previewImage
  }
}
```
or you can just use `previewImage` in your template
`:class="{'file-upload-gallery-item_not-image': !previewImage}"`